.github/workflows/ci.yml

```diff
@@ -52,6 +56,14 @@ jobs:
           cargo clippy --locked --release --all-features --all-targets -- -D clippy::all
           deno lint
 
+      - name: Lint deps
+        if: contains(matrix.os, 'ubuntu')
+        run: |
+          cargo install cargo-machete --locked
+          cargo machete
+          cargo install cargo-udeps --locked
+          cargo +nightly udeps
```

Comment:
Consider putting this after cargo test so that people get faster feedback on failing tests.

Reply:
Good call, swapped.

Reply:
Actually, have to revert it because the tests cause a dep lint failure (because of compile tests :/)